docs/docs/config.md

```diff
@@ -182,6 +182,10 @@ exclude_tables                 | PATTERN                             | ignore up
 blacklist_dbs                  | PATTERN                             | ignore updates AND schema changes from databases (see warnings below) |
 blacklist_tables               | PATTERN                             | ignore updates AND schema changes from tables named like PATTERN (see warnings below) |
 &nbsp;
+**encryption**
+encrypt_data                   | BOOLEAN                             | encrypt the data field of the payload               | false
+encrypt_all                    | BOOLEAN                             | encrypt the entire payload                          | false
```

Comment:
Another way to configure this: `encrypt=[data|all|none]`, default is `none`.

Reply:
Yeah that would probably be better, since you're not allowed to specify both anyway